### ISSUE TITLE
Color indication for requests without status

### ISSUE THREAD

**Issue by tuman18** [janodvarko/harviewer#49]
It will be cool if we could see some additional color indication for requests without status, like in Firebug:
![2015-12-08_1543](https://cloud.githubusercontent.com/assets/4694232/11715745/f94a818a-9f4c-11e5-9359-2cf2b33aef56.png)
Now such requests are not marked at all:
![2015-12-08_1540](https://cloud.githubusercontent.com/assets/4694232/11715748/01b4ac38-9f4d-11e5-87d1-15fbd5610da2.png)

**Comment by gitgrimbo** [comment]
Do you have a sample HAR for this?  Or can you provide a link to the site where you see this behaviour?

I can replicate this somewhat by browsing a site in Chrome 51 with Adblock Plus turned on.  The resources that are blocked have a `response.status` of `0` (using *Copy all as HAR* from dev tools).

I don't have the same luck doing this in Firefox 46 with Adblock Plus turned on (using  *Copy all as HAR* from dev tools).  Possibly the blocked resources don't even appear in Firefox's Network panel.

If this is the same as your situation we can possibly treat `response.status` of `0` as an error and colour it appropriately.

Some extra info about [HTTP status code `0` from Stack Overflow](http://stackoverflow.com/questions/3825581/does-an-http-status-code-of-0-have-any-meaning).

**Comment by tuman18** [comment]
Hi, sorry for the late reply. I have some old saved HARs, although not sure in what exact file is "Aborted" state.
[vk_com (11).zip](https://github.com/janodvarko/harviewer/files/313393/vk_com.11.zip)
[vk_com.zip](https://github.com/janodvarko/harviewer/files/313394/vk_com.zip)

**Comment by gitgrimbo** [comment]
TL;DR - Should be easy enough to add some colour for responses with `response.status==0`.

Details below.

-----

Looking at your vk_com.har, the responses with `response.status==0` have the following `response._error` values  (I think `response._error` is a non-standard HAR property only Chrome sets):

    net::ERR_BLOCKED_BY_CLIENT https://top-fwz1.mail.ru/js/code.js
    net::ERR_BLOCKED_BY_CLIENT https://counter.yadro.ru/hit?rhttps%3A//vk.com/app2164459_99785865;s1920*1080*24;uhttps%3A//vk.com/app2164459_99785865;0.5148746906779706
    net::ERR_BLOCKED_BY_CLIENT https://www.tns-counter.ru/V13a***R%3Ehttps://vk.com/app2164459_99785865*vk_com/ru/UTF-8/tmsec=vksite_total/106731844
    net::ERR_BLOCKED_BY_CLIENT https://sb.scorecardresearch.com/p?c1=2&c2=13765216&c3=&c4=https%3A//vk.com…ttps%3A//vk.com/app2164459_99785865&c15=&cv=2.0&cj=1&rn=0.3027235888876021
    net::ERR_BLOCKED_BY_CLIENT https://an.yandex.ru/system/context.js
    net::ERR_BLOCKED_BY_CLIENT https://openstat.net/cnt.js
    No _error property         https://queuev4.vk.com/im865
    net::ERR_BLOCKED_BY_CLIENT https://top-fwz1.mail.ru/js/code.js
    net::ERR_INSECURE_RESPONSE https://plarium.hs.llnwd.net/v1/geo/client/vk/uc_10_20_0812_635809255632425104.swf
    net::ERR_INSECURE_RESPONSE https://plarium.hs.llnwd.net/v1/geo/preloader/preloader_td_005_mini_ru.jpg
    net::ERR_INSECURE_RESPONSE https://plarium.hs.llnwd.net/v1/sites/portal/games/totaldom/en/game/progress_bar.png

The `ERR_INSECURE_RESPONSE` error seems to be because the browser has not allowed a request where the certificate is invalid (http://stackoverflow.com/questions/22083453/console-errors-failed-to-load-resource-neterr-insecure-response).

The `ERR_BLOCKED_BY_CLIENT` error seems to be because an Ad Blocker has blocked the request (http://stackoverflow.com/questions/23341765/getting-neterr-blocked-by-client-error-on-some-ajax-calls).

**I'm interested to know what the `https://queuev4.vk.com/im865` request is, and why it doesn't have a `response._error`.**

From what I can see, netExport does not include such requests in its HAR export, but Firefox 47 dev tools does (but without the non-standard `response._error` property).

This sample HTML file can reproduce some of the effects you see (if run from a web server, file:// might not work).  The two style links are blocked as they have problems with their certs (Chrome `net::ERR_INSECURE_RESPONSE`).  The script can get blocked by something like Adblock Plus (Chrome `net::ERR_BLOCKED_BY_CLIENT`),

    <!doctype html>

    <head>

    <link type="text/css" rel="stylesheet" href="https://expired.badssl.com/style.css">

    <link type="text/css" rel="stylesheet" href="https://wrong.host.badssl.com/style.css">

    <script src="http://native.sharethrough.com/assets/sfp.js"></script>

    </head>

**Comment by gitgrimbo** [comment]
Can you check http://gitgrimbo.github.io/harviewer/issue-49/ with your HARs to see if this is fixed for you?

**Comment by tuman18** [comment]
Thanks a lot, looks like it is fixed for all my HAR files (I will add a comment if I find some exceptional HAR). Thanks for your time!

**Comment by tuman18** [comment]
Hi, I've used this version http://gitgrimbo.github.io/harviewer/issue-49/ for my HARs, but it isn't working now. Any news about the update?

**Comment by gitgrimbo** [comment]
Hi.  Yeah sorry I've removed that branch as it's now in my master.

http://gitgrimbo.github.io/harviewer/master/

I'll see when these changes can be deployed to the main site, http://www.softwareishard.com/har/viewer/.
